app/resources/serializable_base.rb

```diff
@@ -0,0 +1,11 @@
+class SerializableBase < JSONAPI::Serializable::Resource
```

Comment:
I don't think we need this anymore, this is a jsonapi-rb thing. Now that we're on jsonapi-resources

Reply:
true